package.json

```diff
@@ -104,23 +104,23 @@
     "eslint-plugin-qunit": "^7.3.4",
     "loader.js": "^4.7.0",
     "prettier": "^2.8.7",
-    "qunit": "^2.19.4",
+    "qunit": "^2.20.0",
     "qunit-dom": "^2.0.0",
     "release-it": "^15.10.1",
-    "sinon": "^15.0.3",
+    "sinon": "^16.1.0",
     "typescript": "^5.0.3",
-    "webpack": "^5.77.0"
+    "webpack": "^5.88.2"
   },
   "peerDependencies": {
-    "ember-source": "^3.28.0 || ^4.0.0",
+    "ember-source": "^3.28.0 || ^4.0.0 || >=5.0.0",
     "qunit": "^2.0.0",
-    "sinon": "^15.0.3"
+    "sinon": "^15.0.3 || ^16.0.0"
   },
   "engines": {
-    "node": "14.* || 16.* || >= 18"
+    "node": "16.* || >= 18"
```

Comment:
@hexadecy this would be a breaking change - are sure we actually need this as part of this PR?

Reply:
I'm ok with either approach here. I'm not super gung-ho on keeping Node 14 support when even Node 16 is already EOL at this point as of last month, but if we want to hold off and make that change in a separate release, that's also fine with me.

Reply:
@elwayman02 I would say it's just a matter of another major version bump which is extra churn, but indeed not a problem.

as we already planned v2 conversion, I think we can avoid major just to bump supported Node.js version.

I hope I can spend some time today or tomorrow and I'm sure I can make this PR green without requiring new major.

Reply:
Sounds good!